lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -193,7 +193,7 @@ sub get_data_from_parent_zone {
 sub _check_domain {
     my ( $self, $dn, $type ) = @_;
 
-    my $result = eval {
+    ( $dn, my $result ) = eval {
```

Comment:
Just wondering if this variable name `dn` has a meaning ? I know this is not a new name added in this PR, I was just wondering if someone knew what `dn` stands for. And if this is not explicit enough, why not update the name to something more obvious.

Reply:
To me it's clearly an abbreviation of "domain name". But I agree that it's not great. I'll change it.

Reply:
I think it would help to be consistent on what variable name to use when referring to zone name -- or domain name, if you wish -- to be tested. And to use that variable every time it works, and to reserve it for that.